DxDispatch/src/dxdispatch/Device.h

```diff
@@ -84,6 +87,7 @@ class Device
     Microsoft::WRL::ComPtr<IDMLDevice1> m_dml;
     Microsoft::WRL::ComPtr<IDMLCommandRecorder> m_commandRecorder;
     Microsoft::WRL::ComPtr<ID3D12CommandQueue> m_queue;
+    Microsoft::WRL::ComPtr<ID3D12QueryHeap> m_timestamp_heap;
```

Comment:
nit: inconsistent naming. Only use underscore in `m_` (for member variables). Everything else is pascal case.